README.md

```diff
@@ -116,9 +115,9 @@ Which command produces the following output by default:
 -------------------------------------------
       &nbsp;         mpg   cyl   disp   hp 
 ------------------- ----- ----- ------ ----
-   **Mazda RX4**     21     6    160   110 
+   **Mazda RX4**    21.0    6    160   110 
```

Comment:
We didn't that seems to be my mistake. Turns out `format` works differently on a vector and value
```r
> format(c(21.0, 21.0, 22.8), digits = 4)
[1] "21.0" "21.0" "22.8"
> format(21.0, digits = 4)
[1] "21"
```
I fixed it in https://github.com/Rapporter/pander/commit/9f01b90a6327082dc0e636515d4e075a73e9488c